core/constant_inc.php

```diff
@@ -689,6 +689,9 @@
 # SLIM FRAMEWORK ATTRIBUTES
 define( 'ATTRIBUTE_FORCE_API_ENABLED', 'force_enable_api' );
 
+# Default parent page to use for Manage Configuration sub-menu items
+define( 'DEFAULT_CONFIG_PAGE', 'adm_permissions_report.php' );
```

Comment:
I would suggest `PAGE_CONFIG_DEFAULT`, so in the future, you can add similar constants with same prefix.  This is similar to what we do with other configs/constants.  For example, `MANAGE_CONFIG_ACTION_CREATE`.